vuepress/docs/next/docs/reference/deployment-structure.md

```diff
@@ -99,16 +98,16 @@ The Keycloak image can be customized as part of an Entando implementation. Some
 
 ## Other Key Repositories 
 ### entando-core
-The **entando-core** project is a J2EE application that exposes APIs for the Entando CMS, including the admin console and the portal UI project that performs the server side composition for pages rendered via an Entando Application. Note that only the composition is performed server side. JavaScript code is rendered client-side. The **entando-core** is realized via an instance that includes the 
-WAR files generated from a core build as dependencies. In a default deployment, this is the **entando-de-app**.
+The **entando-core** project is a J2EE application that exposes APIs for the Entando CMS, including the admin console and the portal UI project that performs the server-side composition for pages rendered via an Entando Application. Note that only the composition is performed server side. JavaScript code is rendered client side. The entando-core is realized via an instance that includes the 
```

Comment:
exposes APIs *to* the Entando CMS

composition *of* pages rendered

server-side and client-side should always be hyphenated within the same paragraph (and i think in general, but we haven't added this to the style guide yet)

"the entando-core" is realized"... "realized" is not a great word here; should be more direct and technical. also the last sentence was misinterpreted - the files were not built as dependences, they were generated from a *core build*; the creation of sw is referred to as a build. can rephrase to "...includes the WAR files as dependencies, which are generated from the core build."